code/modules/mob/living/carbon/human/species/yautja/_species.dm

```diff
@@ -220,3 +221,97 @@
 
 /datum/species/yautja/handle_paygrades()
 	return ""
+
+/// Open the Yautja emote panel, which allows them to use their emotes easier.
+/datum/species/yautja/open_emote_panel()
+	var/datum/yautja_emote_panel/ui = new(usr)
+	ui.ui_interact(usr)
+
+/datum/action/yautja_emote_panel
+	name = "Open Emote Panel"
+	action_icon_state = "looc_toggle"
+
+/datum/action/yautja_emote_panel/can_use_action()
+	. = ..()
+	if(!.)
+		return FALSE
+
+	if(!isyautja(owner))
+		return FALSE
+
+	return TRUE
+
+/datum/action/yautja_emote_panel/action_activate()
+	if(!can_use_action())
+		return
+
+	var/mob/living/carbon/human/human_owner = owner
+	var/datum/species/yautja/yautja_species = human_owner.species
+	yautja_species.open_emote_panel()
+
+/datum/yautja_emote_panel
```

Comment:
add a cooldown to using an emote here, please. 1-2 seconds would be enough to prevent lag from making sounds overlap bc someone clicked at a bad time

Reply:
Only reason I didn't include the original cooldown is Yautja already don't have emote cooldowns so they can/do do this already with emote macros.